modules/configuring-firewall.adoc

```diff
@@ -92,10 +96,22 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
 |443, 80
 | Required to access your GCP account.
 
-|Azure
+.4+|Azure
 |`management.azure.com`
 |443, 80
-|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API Reference] in the Azure documentation to determine the endpoints to allow for your APIs.
+|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API reference] in the Azure documentation to determine the endpoints to allow for your APIs.
+
+|`*.blob.core.windows.net`
+|443, 80
+|Required to download Ignition files.
+
+|`login.microsoftonline.com`
+|443, 80
+|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API reference] in the Azure documentation to determine the endpoints to allow for your APIs.
+
+|`www.office.com`
```

Comment:
This is a full URL list I found on a proxy server which is used for a fresh Azure cluster installation:

```
api.openshift.com:443
canary-openshift-ingress-canary.apps.gpei-13.qe.azure.devcluster.openshift.com:443
cdn02.quay.io:443
ci-dv2np-image-registry-us-east-1-aunteqmixxpqypvdqwbmjbiloeix.s3.dualstack.us-east-1.amazonaws.com:443
cloud.redhat.com:443
console-openshift-console.apps.gpei-13.qe.azure.devcluster.openshift.com:443
http://168.63.129.16/?
imageregistrygpei1358nqx.blob.core.windows.net:443
infogw.api.openshift.com:443
login.microsoftonline.com:443
management.azure.com:443
oauth-openshift.apps.gpei-13.qe.azure.devcluster.openshift.com:443
quay.io:443
registry.access.redhat.com:443
registry.ci.openshift.org:443
registry.redhat.io:443
```

The one which is related to Azure maybe:
```
login.microsoftonline.com:443
management.azure.com:443
imageregistrygpei1358nqx.blob.core.windows.net:443
```
I didn't find `www.office.com` there, could you point me where it was used and required?

Reply:
@gpei the customer case comment has :
www.office.com			443, 80		Required to access Azure services and resources. Review the Azure REST API Reference in the Azure documentation to determine the endpoints to allow for your APIs.

Reply:
I don't know where it was used

Reply:
@gpei the `www.office.com` was registered during an OpenShift 4.6.47 installation in a customer environment with a transparent proxy. I think it's part of the redirection in the Azure/Microsft API to retrieve the access token to complete the validation to `management.azure.com`. I could see some connections to `login.microsoftonline.com` and `login.office.com`.